source/stdcpp/vector.d

```diff
@@ -908,6 +904,36 @@ extern(D):
         }
         ///
         extern(C++) void swap(ref vector x);
+        ///
+        void pop_back()
+        {
+            assert(!empty(), "vector.pop_back called on an empty container");
+            this.__destruct_at_end(this.__end_ -1);
+        }
+
+    private:
+        pointer __begin_ =  null;
+        pointer __end_ = null;
```

Comment:
```suggestion
        pointer __begin_;
        pointer __end_;
```

`null` is the default for pointers in D.

Reply:
yess, true... let me make that alterations